src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -222,6 +227,8 @@
                                            (get-in collectible [:preview-url :uri]))
             transaction-type             (:tx-type send-transaction-data)
             estimated-time-min           (reduce + (map :estimated-time route))
+            token-symbol                 (or token-display-name
+                                             (-> send-transaction-data :token :symbol))
```

Comment:
In the bridge flow we aren't passing `token-display-name` so we extract the token value from the transaction data, by doing this we are able to display the token symbol and the token image in the transaction confirmation screen

Reply:
The default value extracts the symbol from token, could this lead to bugs if token is not set (because user is sending a collectible, for example?

Reply:
The worst case is when the data doesn't exist and `token-symbol` would be `nil`, the same as before this fix.

BTW, I tested with collectibles and everyhing is looking good.

Another thing is that this screen seems to need a refactor, mainly to make clear enough what is the data needed in each flow since this screen is being reused multiple times